src/storage/redis_metadata.h

```diff
@@ -203,6 +204,8 @@ class Metadata {
 class HashMetadata : public Metadata {
  public:
   explicit HashMetadata(bool generate_version = true) : Metadata(kRedisHash, generate_version) {}
+
+  bool IsEncodedFieldExpire() const;
```

Comment:
For better understanding, I would rename this function to `IsEncodedFieldHasTTL`/`IsEncodedFieldContainsTTL`.

Reply:
I prefer `IsEncodedFieldHasTTL` because it has fewer letters than `IsEncodedFieldContainsTTL`  :)